packages/mllp-server/.env.example

```diff
@@ -1,10 +1,11 @@
 SENTRY_DSN=your-sentry-dsn
 RELEASE_SHA=your-release-sha
-ENV_TYPE=development
+ENV_TYPE=dev
```

Comment:
Since we're doing this here, can we also update the utils' one, please?

Reply:
addressed